Framework/Core/include/Framework/ASoA.h

```diff
@@ -1722,8 +1722,11 @@ std::tuple<typename Cs::type...> getRowData(arrow::Table* table, T rowIterator,
       return (*mColumnIterator & (static_cast<type>(1) << bit)) >> bit;                                                                                                           \
     }                                                                                                                                                                             \
   };                                                                                                                                                                              \
-  static const o2::framework::expressions::BindingNode _Getter_ { _Label_, typeid(_Name_).hash_code(),                                                                            \
-                                                                  o2::framework::expressions::selectArrowType<MAKEINT(_Size_)>() }
+  static const o2::framework::expressions::BindingNode _Getter_                                                                                                                   \
```

Comment:
any idea why these 3 Framework/Core files were touched? its only formatting, but they should still not be changed by this PR

Reply:
> any idea why these 3 Framework/Core files were touched? its only formatting, but they should still not be changed by this PR

I got formatting issue at the CICD start, so I just used `git clang-format HEAD~1` and `git clang-format origin/dev` (Usually I use `clang-format -style=file -i ...`). Was it wrong decision? Shall I cancel that commit?

Reply:
Usyually `git clang-format HEAD~X` with X being the number of commits you did is fully sufficient. I believe that in your local branch you somehow introduced some merging from `dev` and therefore git thought you touched these files which you did not. Since all your changes are basically in a single commit only you could do (while on your working branch)
```
# update your local repo
git fetch --all
# put your changes on top of dev (in case your remote name is e.g. origin instead of upstream adapt the command)
git rebase upstream/dev
# merge all your changes in a single commit, drop the clang format of framework files
git rebase -i HEAD~3
git push -f
```

Reply:
Ah, yes, I usually develop my code in the dedicated branch, and then I do for O2:
```
git checkout dev
git pull --rebase
git checkout my_branch
git merge dev
```
I guess this is not good schema, isn't it? Should I merge `my_branch` -> `dev` and then push it next time?

Reply:
in O2 there are anyhow no merge commits allowed. The history here is always linear. So I would use rebase instead of merge. Other than that your workflow looks fine

Reply:
I see, okay, then no merging anymore :) What shall I do now? In previous message you attached list of git operations. Am I understood correctly these git commands as  solution for current situation?

Reply:
this would be the easiest to get rid of the unrelated framework changes I think, so yes please

Reply:
@martenole I see that there are still changes in the unrelated framework

Reply:
yes and for some reason your commit "clang" no contains changes both to framework and to your file. So I fear this one you have to fix manually:
```
git reset --soft HEAD~1
# unstage files you don't want to change
git restore --staged <list of framework files>
# restore their previous state
git checkout <list of framework files>
# commit only the changes you want to keep
git commit -m "clang"
git push
```

Reply:
@martenole thanks a lot! Actually I did:
```
git reset --hard main_commit_id
"formatting command"
git commit...
git push --force
```
Now I see only my main commit and formatting of related framework